### DISCUSSION TITLE
will not follow glideslope

### DISCUSSION THREAD

**Discussion by normie555** [flybywiresim/aircraft#4577] [Q&A]
I am new to SimBrief. I have made many flights using the flightplanner in 2020 and a third party planner. I am flying the 320neo Flybywire new aircraft and have no problems with ILS.
Using ILS with SimBrief flightplan however it sees the ILS but will not follow glideslope.
Tried on many airports. same result. Problem probably with flybywire plane.

**Comment by normie555** [comment]
actually worse than I thought.
It totally ignores both localizer and glideslope.

**Reply by yuriks59** [reply]
after the update, the APPR button is not activated and the boarding machine does not work.

**Reply by normie555** [reply]
The APPR button works fine on flightplans other than SimBrief. Are you saying the button stopped only on SimBrief. That's truly odd.

**Comment by ExampleWasTaken** [comment]
There is no update that broke the APPR button. SimBrief doesn't not have any effect on the ILS functionality. 
There are a few things that might cause the aircraft to not pickup the ILS:
1. Make sure the correct ILS frequency is tuned in the `RAD NAV` page! Double check against charts, as the database of the sim is not 100% correct for every airport.
2. Make sure you are below the glideslope before intercepting. Capturing from above is only possible with special procedures!
3. Make sure you intercept the LOC with a maximum of 30°. Intercepting with more than 30° can cause overshooting which can result in intercepting the G/S too late / too high.
4. **Make sure you use either the latest stable or the latest development version!!** Download both of them through the [custom installer](https://api.flybywiresim.com/installer).

**Reply by normie555** [reply]
I am well experienced with ILS.
I repeat ILS works well  with flightplan generated by FS2020.....does NOT work with FlightSim

**Reply by ExampleWasTaken** [reply]
Did you follow the steps I asked you to perform?

**Reply by normie555** [reply]
I  always do. Been doing this for years.

**Comment by normie555** [comment]
Just to make things perfectly clear....... I am connecting to ILS OK........Just will not follow localozer or glideslope

**Reply by ExampleWasTaken** [reply]
What do your FMAs say?

**Reply by normie555** [reply]
FMA?

**Reply by ExampleWasTaken** [reply]
Flight Mode Annunciatiors. The green/blue letters on the upper part of the PFD.

**Reply by normie555** [reply]
They indicate properly with 2020 flightplans showing Glideslope and Localizer at top but as far as I remember didnt show when flying SimBrief flightplan although orange ILS indicators show OK
will run flight again and double check and get back

**Reply by ExampleWasTaken** [reply]
Okay if you check again lookout for these specific things during approach please:

- Each FMA and the respective color. The color is really important!!
- The ILS frequency tuned in the `RAD NAV` page.
- What did the G/S and LOC indicator on the PFD show?

And please tell me the airport you're flying into.

**Reply by normie555** [reply]
I will take screenshots
The airport is CYYZ RWY  33L freq is 110.95

**Reply by normie555** [reply]
![11095](https://user-images.githubusercontent.com/71556913/116013610-f1ef0780-a5e5-11eb-84b0-001f29cadd29.png)
Frequency set OK

![33L](https://user-images.githubusercontent.com/71556913/116013653-37abd000-a5e6-11eb-9674-82dc6f4e94a6.png)
Shows set to RWY 33L

![locandgls](https://user-images.githubusercontent.com/71556913/116013543-94f35180-a5e5-11eb-90fe-1632a304bff6.png)
this shows glideslope and localizer and FREQ OK

![nc](https://user-images.githubusercontent.com/71556913/116013769-b1dc5480-a5e6-11eb-8dd1-247b54d946ee.png)
Shows NOT following glideslope
Also no ref to ILS at top

**Reply by ExampleWasTaken** [reply]
You didn't engage the APPR button

**Reply by ExampleWasTaken** [reply]
![Microsoft Flight Simulator Screenshot 2021 04 26 - 01 55 02 24](https://user-images.githubusercontent.com/58574351/116013827-72d3f080-a632-11eb-8f1a-a0e8c73645c6.png)

You need to press the button labeled `APPR` below the V/S selector. Otherwise the aircraft will intercept neither the LOC nor the G/S. 
Pressing the `ILS` button will only enable the aircraft to find the ILS beam.

**Reply by normie555** [reply]
ARE YOU SERIOUS. OF COURSE I DID!!!!!!!!!

**Reply by ExampleWasTaken** [reply]
No need to scream. The FMAs are what tells you what the A/P is doing. The fact that the ILS is identified but it neither the LOC nor the G/S is armed, tells me that you didn't. If they were armed the FMAs would display `LOC` and `G/S` in blue.

**Reply by normie555** [reply]
As mentioned before the point seems to be that the APPR button does not seem to work with SimBrief flightplan

**Reply by ExampleWasTaken** [reply]
Are you on stable?

**Reply by normie555** [reply]
you dont listen well
no point continuing

**Reply by ExampleWasTaken** [reply]
Ok, you came here because you require help, I am trying to help you, you tell me that you've been doing this for year and that it's the aircraft's fault when nobody ever reported anything similar. If you want help then you need to provide the information the person who tries to help you needs. Otherwise neither of us gets far.

**Reply by ExampleWasTaken** [reply]
I am happy to help you in a kind tone and with the information I need. :)

**Reply by normie555** [reply]
I just completed the same flight using my FS2020 flightplan
everything worked OK

**Reply by ExampleWasTaken** [reply]
Ok, so what version are you using? The development, or the stable?

**Reply by ExampleWasTaken** [reply]
We need to rule out one thing after another, that's why my questions might appear "all over the place" to you.

**Reply by normie555** [reply]
stable

**Reply by ExampleWasTaken** [reply]
Ok thank you. When you did the approach with the flight plan from the sim, did the FMAs show `LOC` and `G/S` in blue?

Also, please remove everything from your community folder including the A32NX. Then download the stable again using the [installer](https://api.flybywiresim.com/installer). This will ensure that you have the latest version, an uncorrupted install and that there are no mod conflicts.

**Comment by normie555** [comment]
Will clear all then run flight again and send screenshot of display
this may take a little while

**Comment by normie555** [comment]
![cup014](https://user-images.githubusercontent.com/71556913/116016049-40a19f00-a5f0-11eb-9c68-15b111f994c0.png)
same flightplan. This time g/s and loc show but did not catch glideslope
will repeat with other flightplan.
Alt was 3500 which is ideal to catch glideslope

**Reply by ExampleWasTaken** [reply]
Okay here we have the answer. If you look at the distance from the threshold in the left bottom corner, you see that you are 3.2 miles away while being on 3500 ft. According to the charts for RWY 33L CYYZ, you would have to be at about 1500 ft (see the image below). On the thread below, when you were using the MSFS flight plan manager you, were 14 nm out when at 3500 ft.

Another thing that you might confuse/didn't know is, you is that you have to select a STAR for your destination manually when importing a SimBrief flight plan, while the MSFS flight plan creator does that for you. This is inline with the real aircraft. This would also explain why you are not in the LOC.

As you can see on this image, at a distance of 3.0 nm you have to be at 1500 ft. As I said earlier capturing the ILS from above requires special procedures and is not possible at all from 3500 ft to 1500 ft. 
![RWY33L_CYYZ](https://user-images.githubusercontent.com/58574351/116114665-da814e80-a6b9-11eb-9642-9def71d36f19.png)

Here is a [video about the MCDU setup](https://youtu.be/c7UJlJMJdfk) by [320 Sim Pilot](https://www.youtube.com/channel/UCAILTiWNai7Y2zsO8ECFxcA). Please note, that the layout and look of the MCDU has since changed a lot to match the real aircraft. The functionality however remained the same.

I am fairly sure that we have fixed this issue! 😃 If you have any more questions be sure to let me know! :)

**Reply by normie555** [reply]
I entered the glideslope feather at 3500 ft  not 3nm. As I said before been doing this for many years

**Reply by ExampleWasTaken** [reply]
Did you ever enter a STAR? Also, a video of your approach, **before** you enter the G/S would be helpfu to catch any mistakes you might make.

**Reply by normie555** [reply]
I will get back to you once I study and apply the video. I was using a different video for reference which missed out some settings

**Reply by ExampleWasTaken** [reply]
Great! 👍

**Reply by normie555** [reply]
THAT SOLVED IT!!!!
The video  I used missed the ARRIVAL settings where the ILS and STARS are set
ILS OK now
Thanks for your help and patience with me.
I am 76 years of age, and inclined to be a little grumpy when frustrated.

**Reply by ExampleWasTaken** [reply]
Great! No worries, that's why I'm here, to help people who get a bit lost in the complex systems of the many different components that make up the A32NX.
Glad we finally got it to work. 👍 😃

**Comment by normie555** [comment]
![cat3](https://user-images.githubusercontent.com/71556913/116017373-41d4cb00-a5f4-11eb-9bcf-4e72de2169a3.png)

![cup015](https://user-images.githubusercontent.com/71556913/116017411-59ac4f00-a5f4-11eb-8cf0-283fd4d9d6b4.png)


![cup016](https://user-images.githubusercontent.com/71556913/116017420-5f099980-a5f4-11eb-9a3c-2558d717ca08.png)


![cup017](https://user-images.githubusercontent.com/71556913/116017430-63ce4d80-a5f4-11eb-8b41-c0c96c0bf079.png)


![cup018](https://user-images.githubusercontent.com/71556913/116017434-68930180-a5f4-11eb-9748-6df4ff48a953.png)

These are the FS2020 flightplan.
as you can see ....perfect

**Reply by normie555** [reply]
I think the simplest answer to this is to ask if you yourself fly a flight importing a flightplan from SinBrief can you catch the glideslope....YES/NO

**Reply by CptBayer** [reply]
I am wondering where the dependency between a flightplan from MS and Simbrief to the ILS functionallity is?! Flightplans don´t tune radios. 

Have you tried flying a pattern at this airport with tuning in all needed settings in the radio? Myself I am flying over 30 years and I can´t follow your issue... hmmm ... I will fly right now a pattern and check this out...

Greets Klaus

**Comment by normie555** [comment]
In this case it does. When you select the ILS in the flightplan from the aircraft it populates the frequency of the ILS

**Reply by CptBayer** [reply]
ah ok. Now I know what you mean! The autotuned ILS in the RAD/NAV due to the imported flightplan is a normal thing. Correct. So with your imported flightplan the frequency is in the RAD/NAV visible but your aircraft is not catching the glidepath .... 

I did a pattern without flightplan.... take a look at my video. You can download it via the following link from uploaded.net. Tomorrow I will try it with a simbrief flightplan with a SID STAR combination from CYYZ to CYYZ .... 

http://ul.to/c1xb2a4n or youtube

https://youtu.be/_7KEOm64c6A

Greets Klaus

**Reply by ExampleWasTaken** [reply]
No need. We figured out what was wrong. :)

**Reply by CptBayer** [reply]
ah cool. What was the issue?

**Reply by ExampleWasTaken** [reply]
They didn't know that you have to manually select SIDs/STARs when importing a SimBrief flight plan, as they are new to SimBrief. 
Me not thinking about that to be an option for possible error, plus a few misunderstandings led to the 40+ messages.

**Reply by CptBayer** [reply]
hahaha... ok .... simple solution.... to simple for people who know how to do it in the right way :-p.....  there is a nice saying for such a situation: to much trees in front of you to recognize the forrest  ...... 


cheers ....
